apis/infix-en_GB/atrium-api-infix-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/infix/en_GB/AnyAssertionsSpec.kt

```diff
@@ -76,12 +76,12 @@ class AnyAssertionsSpec : ch.tutteli.atrium.specs.integration.AnyAssertionsSpec(
         a1b notToBeNull {}
 
         a1 and o toBe 1
```

Comment:
Let me guess, we cannot make this one also `it`?

Reply:
that's exactly the issue